herbert_core/utilities/general/is_integer_id.m

```diff
@@ -16,6 +16,13 @@
 %           If not, ix is such that id_in(ix) is sorted in ascending order
 %   perm    True if id is a permutation of the integers 1:numel(id)
 
+% empty id_in will crash the all(id>=1) below
+if isempty(id_in)
+    ok = false;
+    mess = 'HERBERT:is_integer_id:invalid_argument';
```

Comment:
Strictly this is an identifier not a message, but irrelevant. 

Worth changing to proper error handling or too deeply intertwined with other code?

Reply:
to get the Herbert/Horace merge done. leave this for now.